data/virtualization/agama_virt_auto/sle_virt_default_staging.jsonnet

```diff
@@ -0,0 +1,98 @@
+local repo = '{{INCIDENT_REPO}}';
+local urls = if repo != '' then std.split(repo, ',') else [];
+{
+  product: {
+    id: '{{AGAMA_PRODUCT_ID}}',
+    registrationCode: '{{SCC_REGCODE}}',
+    addons: [
+      {
+        id: 'PackageHub',
```

Comment:
Hi Roy,
Do we have any tests specifically needing packagehub? Packagehub is for unspported pkgs

Reply:
I don't know where to use it, just copy from core team.

Reply:
if doesn't need it, I will remove it.

Reply:
Ok, let's remove it now. If needed, we can add it back later.

Reply:
removed. and smoke test works well.
https://openqa.oqa.prg2.suse.org/tests/19447369#details